libpromises/evalfunction.c

```diff
@@ -7062,6 +7062,27 @@ static FnCallResult FnCallStringSplit(ARG_UNUSED EvalContext *ctx,
 
 /*********************************************************************/
 
+static FnCallResult FnCallStringReplace(ARG_UNUSED EvalContext *ctx,
+                                        ARG_UNUSED const Policy *policy,
```

Comment:
`const` applies to the left, unless it is the first word, then it applies to the right. Since you're using the ARG_UNUSED macro here, I think it might be more clear to move it past `Policy`. Not a big deal, this should work.